web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-media.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-pvtrqq.php, but found phpcs-scan-pVTRQq.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).